spec/system/admin/sdg/managers_spec.rb

```diff
@@ -0,0 +1,94 @@
+require "rails_helper"
+
+describe "Admin SDG managers" do
+  let!(:user) { create(:user) }
+  let!(:sdg_manager) { create(:sdg_manager) }
+
+  before do
+    login_as(create(:administrator).user)
+    visit admin_sdg_managers_path
+  end
+
+  scenario "Index" do
+    expect(page).to have_content sdg_manager.name
+    expect(page).to have_content sdg_manager.email
+    expect(page).not_to have_content user.name
+  end
+
+  scenario "Create SDG Manager", :js do
+    fill_in "name_or_email", with: user.email
+    click_button "Search"
+
+    expect(page).to have_content user.name
+    click_link "Add"
```

Comment:
I think we usually add a blank line between expectations and actions :thinking:.